packages/pigeon/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 4.2.3
+
+* [kotlin] Fixes Kotlin generation error when using the generate sync host api
```

Comment:
This is missing the period required by the style guide.

I'm also having trouble parsing the last part of this sentence. I'm not sure what "generation error when using the generate [generated?] sync host api" means; generation doesn't happen when using the API. This also doesn't actually explain what the error is, just that there is one.

Reply:
done